lib/src/debugger.dart

```diff
@@ -32,11 +32,13 @@ class WipDebugger extends WipDomain {
 
   Future<WipResponse> resume() => sendCommand('Debugger.resume');
 
-  Future<WipResponse> stepInto() => sendCommand('Debugger.stepInto');
+  Future<WipResponse> stepInto({Map<String, dynamic> params}) =>
+      sendCommand('Debugger.stepInto', params: params);
 
   Future<WipResponse> stepOut() => sendCommand('Debugger.stepOut');
 
-  Future<WipResponse> stepOver() => sendCommand('Debugger.stepOver');
+  Future<WipResponse> stepOver({Map<String, dynamic> params}) =>
```

Comment:
Does `stepOut()` not take the same skipList params as stepInto and stepOver?

Reply:
Nope: https://chromedevtools.github.io/devtools-protocol/tot/Debugger/#method-stepOut